code/modules/jobs/job_types/agent.dm

```diff
@@ -95,7 +95,8 @@
 	name = "Agent"
 	jobtype = /datum/job/agent
 
-	belt = /obj/item/pda/security
+	belt = /obj/item/storage/belt/ego
```

Comment:
What's the point of making it spawn on people? I'd suggest putting it on the maps, or adding a vendor for all this additional equipment.

Reply:
It was mostly for testing because this was TMed for a while. I will add to the maps.